HyperVExtension/src/HyperVExtension/Constants.cs

```diff
@@ -7,5 +7,5 @@ internal sealed class Constants
 {
     public const string WindowsThumbnail = "ms-appx:///HyperVExtension/Assets/hyper-v-windows-default-image.jpg";
 
-    public const string ExtensionIcon = "ms-appx:///HyperVExtension/Assets/hyper-v-provider-icon.png";
+    public const string ExtensionIcon = "ms-resource://Microsoft.Windows.DevHome.Dev/Files/HyperVExtension/Assets/hyper-v-provider-icon.png";
```

Comment:
Do we need a comment or a link to documentation about the format of this string? (developers may be looking at this code for an example how to create a DevEnv extension).